modules/storage-aws-s3/module.go

```diff
@@ -76,6 +77,9 @@ func (m *StorageS3Module) MetaInfo() (map[string]interface{}, error) {
 	metaInfo := make(map[string]interface{})
 	metaInfo["endpoint"] = m.config.Endpoint()
 	metaInfo["bucketName"] = m.config.BucketName()
+	if len(m.config.RootName()) > 0 {
```

Comment:
Try this if `m.config.RootName() != ""  {`

It is performant and more idiomatic

Reply:
This is what I had originally, but if you look elsewhere in the package, `len()` is used to check for empty strings, so I was attempting to remain consistent with the rest of the package

Reply:
fixed